src/Store/EmptyStore.php

```diff
@@ -2,21 +2,8 @@
 
 namespace Auth0\SDK\Store;
 
-/*
- * This file is part of Auth0-PHP package.
- *
- * (c) Auth0
- *
- * For the full copyright and license information, please view the LICENSE file
- * that was distributed with this source code.
- */
-
-
-
 /**
  * This class is a mockup store, that discards the values, its a way of saying no store.
- *
- * @author Auth0
  */
 class EmptyStore implements StoreInterface
```

Comment:
Assuming state should already be an enforced check, what is the reason to keep this store implementation around?

Reply:
Not sure but I'll evaluate in another PR. State can currently be skipped in this SDK